### PR TITLE
dorepl: allow passing a URL for the API to target

### PR DESCRIPTION
**Pull request by aybabtme**

### CODE REVIEW

**Comment by iheanyi** [comment]
LGTM!

**Comment by lxfontes** [comment]
👕 🇮🇹

File: cmd/dorepl/main.go [hunk 1]
```diff
@@ -60,10 +61,16 @@ func main() {
 		flag.PrintDefaults()
 		log.Fatalf("At this time, the REPL requires you to provide an API token")
 	}
-
-	gc := godo.NewClient(oauth2.NewClient(oauth2.NoContext,
+	var opts []godo.ClientOpt
+	if *apiURL != "" {
+		opts = append(opts, godo.SetBaseURL(*apiURL))
+	}
+	gc, err := godo.New(oauth2.NewClient(oauth2.NoContext,
```
Comment (nanzhong) [inline cmd/dorepl/main.go]: What's the diff between `godo.New` and `godo.NewClient`?

Comment (aybabtme) [reply cmd/dorepl/main.go]: @nanzhong `New` takes the func. opts

**Pull request merged**